README.md

```diff
@@ -37,8 +42,9 @@ Additional description about the project and its features.
 
 Contributions, issues, and feature requests are welcome!
 
-
-
 ## Show your support
 
 Give a ⭐️ if you like this project!
+
+## Copyright
+ © Copyright Cindy Shin Korea Republic, 2021
```

Comment:
[OPTIONAL]

- [ ] Please use acknowledgment instead of copyright. Copyright is giving the impression you simply reproduced or copied the whole project. Under the Creative Commons license, acknowledging the designer of the template you used is sufficient.